beacon-chain/blockchain/van_process_pending_blocks.go

```diff
@@ -222,13 +224,20 @@ func (s *Service) waitForConfirmation(b interfaces.SignedBeaconBlock) error {
 
 // verifyPandoraShardInfo
 func (s *Service) verifyPandoraShardInfo(parentBlk, curBlk interfaces.SignedBeaconBlock) error {
+	var parentPanShards []*ethpb.PandoraShard
+
 	// For slot #1, we don't have shard info for previous block so short circuit here
 	if curBlk.Block().Slot() == 1 {
```

Comment:
here is still breakage of `law of demeter`

Reply:
```go
if curBlk == nil || curBlk.IsNil() || curBlk.Block().IsNil() {
		return $PUT_YOUR_ERROR_HERE
	}
```

Reply:
fixed